dlpy/network.py

```diff
@@ -168,10 +168,6 @@ def compile(self):
         rt = self._retrieve_('deeplearn.buildmodel',
                              model=dict(name=self.model_name, replace=True), type=self.model_type)
 
-        if not all(x.can_be_last_layer for x in self.output_layers):
```

Comment:
After the change, those layers still can be added to the model table. CAS side will finally determine whether a head layer is redundant and whether the model can be trained given dataspecs.

Reply:
so we wont be complaining due to inproper model architecture. it is kind of a breaking change. how about we convert the error to note and explain this change.

Reply:
I thought of converting the part into a note. However, it will cause an attribute error when a model built by load() is compiled. For example, if the model is loaded via sashdat, the model won't have attributes output_layers defined in Network.__init__(). One way to solve the no attribute issue is to find out all of output layers. Adding the following line:
`model.output_layers = [l for l in model.layers if l.can_be_last_layer]`
in **from_sashdat** and **from_table** before model object is returned. Is it okay?

Reply:
I am kind of undecided. let see what @sasxianhu thinks on this

Reply:
I actually prefer to let the server decide whether it could run or not since in some cases, we do not have any output layers at all. For instance, using astore to extract features.